src/main/java/nl/knaw/dans/ingest/core/deposit/DepositReaderImpl.java

```diff
@@ -117,6 +127,58 @@ Deposit mapToDeposit(Path path, Path bagDir, Configuration config, Bag bag) {
         return deposit;
     }
 
+    private OriginalFilePathMapping getOriginalFilePathMapping(Path bagDir) throws IOException {
+        var originalFilepathsFile = bagDir.resolve("original-filepaths.txt");
+
+        if (Files.exists(originalFilepathsFile)) {
+            var lines = Files.readAllLines(originalFilepathsFile);
+            var mappings = lines.stream().map(line -> {
+                    var parts = line.split(" {2}", 2);
```

Comment:
* In `dd-validate-dans-bags` I only see examples of this file with a single space as separator.
* I don't see a new unit test with `original-filepaths.txt`

Reply:
* The code in dd-sword2 generates original-filepaths.txt with double spaces, so that would mean the dd-validate-dans-bag examples are wrong.
* Good point

Reply:
I don't consider it good practice that validation accepts file formats that can't be processed, whichever way they are currently(!) produced. The validator even accepts tabs. So please use the same rules to split the lines.
https://github.com/DANS-KNAW/dd-validate-dans-bag/blob/a7f05197b47dee9214f8ce30d3568e92a7ac5013/application/src/main/java/nl/knaw/dans/validatedansbag/core/service/OriginalFilepathsServiceImpl.java#L49-L51

On the other hand is your method of splitting the file into lines more elegant an reliable.

A side note: Java' `\s` doesn't properly cover all Unicode white-space https://stackoverflow.com/questions/4731055/whitespace-matching-regex-java

Reply:
You could have linked to the official spec and have the specification be the leading entity here, not the validator: https://dans-knaw.github.io/dans-bagit-profile/versions/1.0.0/#33-original-filepathstxt

Do you really think the \s unicode issue is ever going to be an issue? If so, please add a ticket to jira to fix it.